demo/combo-chart/combo-chart.component.ts

```diff
@@ -0,0 +1,514 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation,
+  Output,
+  EventEmitter,
+  ChangeDetectionStrategy,
+  ViewChild,
+  HostListener,
+  OnInit,
+  OnChanges,
+  ContentChild,
+  TemplateRef
+} from '@angular/core';
+import { PathLocationStrategy } from '@angular/common';
+import {
+  trigger,
+  state,
+  style,
+  animate,
+  transition
+} from '@angular/animations';
+
+import { NgxChartsModule, BaseChartComponent, LineComponent, LineSeriesComponent } from '../../../ngx-charts';
```

Comment:
I am getting an error here. It should be
`import { NgxChartsModule, BaseChartComponent, LineComponent, LineSeriesComponent } from '../../src';`